partiql-eval/src/eval.rs

```diff
@@ -326,7 +326,11 @@ impl Evaluable for EvalProject {
             let mut t = Tuple::new();
 
             self.exprs.iter().for_each(|(alias, expr)| {
-                t.insert(alias.as_str(), expr.evaluate(&v_as_tuple, ctx));
+                let evaluated_val = expr.evaluate(&v_as_tuple, ctx);
+                if evaluated_val != Missing {
+                    // Per section 2 of PartiQL spec: "value MISSING may not appear as an attribute value
+                    t.insert(alias.as_str(), evaluated_val);
+                }
```

Comment:
Previous behavior kept projected tuple values that evaluated to `MISSING`. Changed to now omit those attribute pairs.

Perhaps a better modeling, would be as the spec mentions in [section 6.3](https://partiql.org/assets/PartiQL-Specification.pdf#subsection.6.3) -- that is modeling is a `SELECT VALUE`. This modeling would likely involve changing the logical to evaluation plan in `plan.rs`.